packages/nouns-contracts/tasks/deploy-local.ts

```diff
@@ -105,22 +117,28 @@ task('deploy-local', 'Deploy contracts to hardhat')
       NounsDAOExecutor: {
         args: [expectedNounsDAOProxyAddress, args.timelockDelay],
       },
-      NounsDAOLogicV1: {
+      NounsDAOLogicV2: {
         waitForConfirmation: true,
       },
-      NounsDAOProxy: {
+      NounsDAOProxyV2: {
```

Comment:
@eladmallel same as above, should this stay V1 until we upgrade?

Reply:
in this case I'd rather keep it on v2, since we're very close to deploying it, and we're not really relying on local runs to test the upgrade process. wdyt?

Reply:
I don't know when we'll actually do the upgrade.
I think keep it on V1 until then can be less confusing for potential forkers